src/Timer/Timers.php

```diff
@@ -73,6 +73,11 @@ public function isEmpty()
 
     public function tick()
     {
+        // hot path: skip timers if nothing is scheduled
+        if (!$this->schedule) {
+            return;
+        }
```

Comment:
While this improves synthetic benchmarks, real world apps will likely perform a bit worse due to the additional evaluation of the if, no?

Reply:
Shows no measurable impact even in worst-case scenarios:

> As a worst-case scenario, you can also execute `time php examples/94-benchmark-timers-delay.php` which constantly schedules a single timer for the next loop iteration. With or without my changes, this shows `7.1s` on my machine, so this confirms my changeset does not negatively impact performance.